src/test/java/org/isf/medicalstock/Tests.java

```diff
@@ -1372,6 +1392,101 @@ void testDeleteLastMovementWithPriorMovements(boolean in, boolean out, boolean t
 		assertThat(followingMovement2).isNotPresent();
 	}
 
+	@Test
```

Comment:
Is there a reason these tests are not set up to be `@ParmeterizedTest`s?  Even if there should be no interaction running them that way would prove it, wouldn't it?

Reply:
yes, I was in doubt about this indeed... let me add